templates/demo-store/app/root.tsx

```diff
@@ -56,8 +56,8 @@ export async function loader({request, context}: LoaderArgs) {
   return defer({
     isLoggedIn: Boolean(customerAccessToken),
     layout,
-    selectedLocale: context.storefront.i18n,
-    cart: cartId ? getCart(context, cartId) : undefined,
+    selectedLocale: storefront.i18n,
+    cart: cart.get(),
```

Comment:
get what exactly?

Reply:
Do you have a better name for this?

In context of a cart instance that performs cart mutation queries, `cart.get()` returns a visitor's cart base on the cart-id in the request header.

`cart.getCart()` is seems redundant naming. We have other methods like:

* `cart.addLines(lines)`
* `cart.updateNote(note)` ... etc